crowdsource/tests/test_forms.py

```diff
@@ -0,0 +1,137 @@
+from django.test import TestCase
+from crowdsource.forms import IssueForm, ResolverForm
+from opencivicdata.core.models import Jurisdiction, Person, Division
+
+
+class BaseFormTestCase(TestCase):
+
+    def setUp(self):
+        division = Division.objects.create(
+            id='ocd-division/country:us', name='USA')
+        self.jur = Jurisdiction.objects.create(
+            id="ocd-division/country:us/state:mo",
+            name="Missouri State Senate",
+            url="http://www.senate.mo.gov",
+            division=division,
+        )
+        self.person = Person.objects.create(name='Name')
+
+
+class IssueFormTestCase(BaseFormTestCase):
```

Comment:
as mentioned before, form tests here are testing the wrong things

Reply:
Agreed, would love to hear, what kind of tests should be written for forms.